client/less/main.less

```diff
@@ -1116,6 +1116,78 @@ code {
   }
 }
 
+// diagonal sash
```

Comment:
This code should probably go in a seperate `shop.less` file.  We need to keep our code separated for organizational purposes..

Also, if the classes are shop specific they should probably have a shop specific name.